modules/covid_research/app/controllers/covid_research/volunteer/submissions_controller.rb

```diff
@@ -10,8 +10,11 @@ class SubmissionsController < BaseController
       INTAKE_EMAIL_TEMPLATE_NAME = 'signup_confirmation.html.erb'
       UPDATE_EMAIL_TEMPLATE_NAME = 'update_confirmation.html.erb'
 
+      INTAKE_SCHEMA = 'COVID-VACCINE-TRIAL'
+      UPDATE_SCHEMA = 'COVID-VACCINE-TRIAL-UPDATE'
```

Comment:
just being nitpicky here, but looks like you are only using this vars once. why not just drop the strings into the FormService methods directly?

Reply:
I like to put string constants like these into constant variables at the top of the file so if/when we need to update the names it is very clear where that update should occur.  In this case it is a pretty simple file so probably not needed.